src/config/dynamic-theme-fixes.config

```diff
@@ -708,6 +708,29 @@ CSS
 
 ================================
 
+google.*
```

Comment:
Hi @Rakleed, please have a look at `www.google.*` and `www.google.*.*` and move your fixes there.